rules/windows/persistence_dontexpirepasswd_account.toml

```diff
@@ -23,6 +23,47 @@ index = ["winlogbeat-*", "logs-system.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "Account configured with never Expiring Password"
+note = """## Triage and analysis
+
+### Investigating Account configured with never Expiring Password
+
+Active Directory offers the capability of setting a user password to don't expire. This is a bad practice and can expose
+your environment to vulnerabilities and weaken the environment's security posture, mainly when these accounts are privileged.
+
+Usually, the reason to configure this setting is to use this user account to act as a service account. Attackers can
+abuse these accounts to persist in the domain and maintain long-term access using compromised accounts with this property
+set.
```

Comment:
```suggestion

```

Reply:
Probably don't need to specify that the compromised accounts have this setting enabled since this info is mentioned at the start of the paragraph.